test-cases/longevity/longevity-nosqlbench-3h.yaml

```diff
@@ -14,7 +14,7 @@ gce_instance_type_loader: 'e2-standard-4'
 
 ssh_transport: 'libssh2'
 
-nemesis_class_name: 'ChaosMonkey'
+nemesis_class_name: 'NonDisruptive'
```

Comment:
Why not use sysiphus monkey with filter by non-disruptive flag?

Reply:
@aleksbykov As far as I know, Sisyphus only picks up properties set to `True`, so only ones that are disruptive. For it to pick up non-disruptive, we'd have to either add a `non_disruptive` flag or rework the filtering logic for `nemesis_include_filter`.

Reply:
This was a quick fix because we didn't have a way to filter for non disruptive with the current way it's implemented and I didn't have time to drill into that. (I'm trying to avoid having another flag saying "non-disruptive: true".